commit c656823cbd26c80cb5100b7b44b329069b96d371 (refs/changes/40/3340/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-08T15:16:11-08:00 (1 year, 8 months ago)
    
    TRITON-119 fix logged var, CREATED_MACHINE_UUID, in MG prep_dataset_in_jpc.sh

diff --git a/tools/prep_dataset_in_jpc.sh b/tools/prep_dataset_in_jpc.sh
index b661929..2561723 100755
--- a/tools/prep_dataset_in_jpc.sh
+++ b/tools/prep_dataset_in_jpc.sh
@@ -298,7 +298,7 @@ for tb_info in ${tarballs}; do
     uncompress=gzcat
   fi
 
-  echo "Copying tarball '${tb_tarball}' to zone '${uuid}'."
+  echo "Copying tarball '${tb_tarball}' to zone '${CREATED_MACHINE_UUID}'."
   if [[ ${tb_sysroot} == "/" ]]; then
     # Special case: for tb_sysroot == '/' we presume these are fs-tarball
     # style tarballs with "/root/..." and "/site/...". We strip
